src/layout/NavigationButtons/NavigationButtonsComponent.tsx

```diff
@@ -143,6 +156,15 @@ function NavigationButtonsComponentInner({
       await navigateToPage(returnToView, { skipAutoSave: true });
     });
 
+  const onClickBackToComponent = () =>
+    performProcess('backToComponent', async () => {
```

Comment:
These things as well should be renamed to something that indicates we're going back to another page, not a component.